rasa/shared/core/events.py

```diff
@@ -1473,6 +1473,14 @@ def __init__(
 
         super().__init__(timestamp, metadata)
 
+    def __members(self) -> Tuple[Optional[Text], Optional[Text], Text]:
```

Comment:
Just curious: Did you mean to do `__members__` instead of `__members`?
Ref: https://docs.python.org/3/library/enum.html#iteration

Reply:
👍